src/basic/chase.c

```diff
@@ -583,6 +583,37 @@ int chase(const char *path, const char *root, ChaseFlags flags, char **ret_path,
         return r;
 }
 
+int prefix_path_from_chaseat(const char *path, const char *root, char **ret) {
```

Comment:
Shouldn't we use this is chase() as well?

Reply:
I do not think so, calling `path_make_absolute_cwd()` and `path_simplify()` is not necessary in `chase()`, as it is already done at the beginning of `chase()` in slightly different and complex conditions.